bgpd/bgp_attr.c

```diff
@@ -1194,7 +1194,6 @@ void bgp_attr_unintern(struct attr **pattr)
 	if (attr->refcnt == 0) {
 		ret = hash_release(attrhash, attr);
 		assert(ret != NULL);
-		bgp_attr_extra_free(attr);
```

Comment:
@ton31337 could you please explain this change? Why shouldn't we free the extra data before freeing the attr?

Reply:
Okay, I understand now that the pointer to extra is copied to `tmp` and needed for subsequent call to `bgp_attr_unintern_sub`. But then you must add freeing to `bgp_attr_unintern_sub`, right?